<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>association</class>
 <widget class="QDialog" name="association">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Association by Criterion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="form_area">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="criterion_label">
            <property name="text">
             <string>Criterion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="criterion_1">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sample ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Process Order</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data Type</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="criterion_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sample ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Process Order</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data Type</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="criterion_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sample ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Process Order</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data Type</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="criterion_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sample ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Process Order</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Data Type</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="condition_label">
            <property name="text">
             <string>Condition</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="condition_1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Same</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Different</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Value</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="condition_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Same</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Different</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Value</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="condition_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Same</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Different</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Value</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="condition_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>&lt;&lt; select &gt;&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Same</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Different</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Value</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>12</number>
          </property>
          <item>
           <widget class="QLabel" name="value_label">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="value_1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="value_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="value_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="value_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="consecutives">
        <property name="text">
         <string>Group only consecutive commands.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontal_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_save">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_load">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_accept">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
